scripts/lint.sh

```diff
@@ -4,7 +4,7 @@ set -e
 echo "Linting..."
 
 check_files=$(git ls-files \
-              | grep -v -E ".jpg|.svg|3rdparty" | cat)
```

Comment:
I think this was intentional for alignment

Reply:
ACK - fixed